servers/src/main/java/tachyon/master/file/journal/AddMountPointEntry.java

```diff
@@ -36,7 +37,9 @@
    * @param tachyonPath the Tachyon path
    * @param ufsPath the UFS path
    */
-  public AddMountPointEntry(TachyonURI tachyonPath, TachyonURI ufsPath) {
+  @JsonCreator
+  public AddMountPointEntry(@JsonProperty("tachyonPath") TachyonURI tachyonPath,
```

Comment:
The getter returns a string for `tachyonPath` in serialization, but how can it be deserialized into `TachyonURI`?